src/monika-config-schema.json

```diff
@@ -302,6 +302,13 @@
           "body": {
             "$ref": "#/definitions/body"
           },
+          "followRedirects": {
+            "title": "Follow Redirects",
+            "description": "The request follows redirects as many times as specified here",
+            "type": "integer",
+            "examples": ["21"],
+            "default": "21"
```

Comment:
just curious, why the default is 21. feels like a magic number without any comment

Reply:
It follows the default value for the [follow-redirects flag](https://monika.hyperjump.tech/guides/cli-options#follow-redirects).